### PR TITLE
Send feature isolate data

### PR DESCRIPTION
**Pull request by banklesss**

### CODE REVIEW

File: src/TxHistory/TxHistoryNavigator.tsx [hunk 1]
```diff
@@ -98,42 +86,20 @@ export const TxHistoryNavigator = () => {
...
-          )}
+          {() => <AssetSelectorScreen balance={balance} />}
         </Stack.Screen>
 
         <Stack.Screen //
           name="address-reader-qr"
           options={{title: strings.qrScannerTitle}}
         >
-          {() => <AddressReaderQR setQrAmount={setAmount} setQrReceiver={setReceiver} />}
+          {() => <AddressReaderQR />}
```
Comment (stackchain) [inline src/TxHistory/TxHistoryNavigator.tsx]: component={AddressReaderQR}

**Review by stackchain — CHANGES_REQUESTED** [review]

File: src/TxHistory/TxHistoryNavigator.tsx (hunk 1 above)
Comment (banklesss) [reply src/TxHistory/TxHistoryNavigator.tsx]: Done. Thanks!

File: src/Send/SendScreen/SendScreen.stories.tsx [hunk 2]
```diff
@@ -1,50 +1,38 @@
...
-        receiver=""
-        setReceiver={action('setReceiver')}
-      />
-    )
-  })
+  .add('Default', () => <SendScreenTest isSendAll={false} />)
+  .add('SendAll', () => <SendScreenTest isSendAll={true} />)
+
+const SendScreenTest = ({isSendAll}: Props) => {
+  mockWallet.subscribe = () => undefined
```
Comment (wolverineks) [inline src/Send/SendScreen/SendScreen.stories.tsx:15]: avoid mutation

Comment (stackchain) [reply src/Send/SendScreen/SendScreen.stories.tsx:15]: https://github.com/Emurgo/yoroi-mobile/pull/2099/files#diff-f23eed79018887d6a48987da7c5dad5e046e85e1ed98d15964c483d70c3252c5R17

**Pull request merged**